files/system/usr/share/ublue-os/just/70-secureblue.just

```diff
@@ -672,3 +672,7 @@ audit-secureblue:
         done
     fi
 
+#Debug pastebin for issue reporting
+debug-info: 
```

Comment:
maybe debug-dump or debug-pastebin so users know this is going to be published to fpaste?

Reply:
I am tend towards debug-info but I followed the example from Device-info, that does not require confirm and is not private by default, since I plan to add a "confirm upload to pastebin" before uploading it should not be a surprise for the user, what do you think?

Reply:
yeah seems fine